CMakeLists.txt

```
@@ -58,7 +58,8 @@ if(${CMAKE_HOST_WIN32})
 	endif()
 endif()
 
-target_link_libraries(rimage PRIVATE crypto)
+find_package(OpenSSL REQUIRED)
+target_link_libraries(rimage PRIVATE crypto OpenSSL::SSL)
```

Smaller dependency enough:

```suggestion
target_link_libraries(rimage PRIVATE crypto OpenSSL::crypto)
```

(OpenSSL::SSL depends on OpenSSL::crypto)